scu-de/src/fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2/version.json

```
@@ -0,0 +1,6 @@
+{
+  "version": "1.3.63",
```

```suggestion
  "version": "1.3.64",
```
If I'm not mistaken 😁